### ISSUE TITLE
Add ability to reset password

### ISSUE THREAD

**Issue by mohit2530** [earmuff-jam/fleetwise#140]
## Acceptance Criteria

As users might forget their password, use the secret key users create when they first create their profile to reset their password. If the user also forgets the secret key then they are out of luck. Contact support in that case, but they would need to present identification at this point - virtually and that is not setup yet neither is in the pipeline just yet.

1. Add secret key token during create profile. This secret key token must be stored like the pwd in the app. Secret key is just a security question and answer. Allow users the ability to ONLY reset the password if they match the secret key. Users are not able to view any parts of the app with the secret key ( security question ).
2. Limit reset of password to only 3 times per council. This must be mandated to protect from duplicate account and scam accounts. This clause should be provided to the audience to read and understand if possible.
3. Resetting the password must not use the previous password and must be a new value.
4. Validate the token timeout for the audience.

**Comment by mohit2530** [comment]
This is blocked by https://github.com/earmuff-jam/FleetWise/issues/315 since we need to configure mail service before we can attempt to fix this. Due to this reason, we postponed the feature to V3 Milestone.

**Comment by earmuff-jam** [comment]
Closed in favor of #342 #343
